plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte

```diff
@@ -87,7 +87,7 @@
 
   $: _id &&
     _class &&
-    queryClient.query(
+    queryClient.query<Issue>(
```

Comment:
Pardon for unrelated changes, made them while trying to understand how the code works, and minor TS warnings were distracting me

Reply:
It s fine to clean warnings.

Reply:
@anotherpit just curious, did you see this warning during transpilation? I previously saw them reported as errors but only in IDEA because of their TS service was incapable of properly parsing the construction. It was all good during the build and in VSCode w/o specifying the type.
See VSCode derives the type from the first parameter:
![image](https://github.com/hcengineering/platform/assets/20221821/06b78aec-0e5e-4f98-8fdb-fd2df3d6c101)